spec/views/users/users_index_spec.rb

```diff
@@ -40,5 +40,10 @@
       expect(page).to have_content(user_post.comments.count)
       expect(page).to have_content(user_post.likes.count)
     end
+
```

Comment:
- [ ] As the previous reviewer suggested kindly consider adding tests for the `User` index page by modifying these files and removing unrelated tests. here are the required tests for the user index page 👇🏼. You can see is the before method on line 10 you're visiting the posts index page instead ` visit "/users/#{@user.id}/posts"`, it should be   `visit "/users"`

![](https://user-images.githubusercontent.com/103386183/255639827-38ab17e0-9f2d-47f3-8968-1b0d66d5bee5.png)